forte/data/entry_type_generator.py

```diff
@@ -10,6 +11,15 @@ def _get_type_attributes():
     return type_attributes
 
 
+def _get_entry_attribute_by_class(input_entry_class: str):
+    modname, _, clsname = input_entry_class.rpartition(".")
```

Comment:
Maybe we should use the utilities [here](https://github.com/asyml/forte/blob/master/forte/utils/utils.py#L79) to get class and import, which would help future debugs:

Reply:
OK, In new PR will modify this.